client/lib/posts/post-edit-store.js

```diff
@@ -201,11 +188,6 @@ function set( attributes ) {
 }
 
 function normalize( post ) {
-	var categoryIds = getCategoryIds( post );
-	if ( categoryIds ) {
-		post.category_ids = categoryIds;
```

Comment:
There's a lingering reference to this (and other?) attribute(s) in `actions.js`:

https://github.com/Automattic/wp-calypso/blob/4a4f18c/client/lib/posts/actions.js#L91-L97